pkg/cloud/utils_test.go

```diff
@@ -31,16 +31,16 @@ func TestEqualResourceID(t *testing.T) {
 		b *ResourceID
 	}{
 		{
-			a: &ResourceID{"some-gce-project", "projects", nil},
-			b: &ResourceID{"some-gce-project", "projects", nil},
+			a: &ResourceID{"some-gce-project", meta.APIGroupCompute, "projects", nil},
```

Comment:
Seems like we don't test for the network services compute prefix?

Reply:
Good call - added test for NetworkServices